scripts/library.sh

```diff
@@ -417,6 +417,22 @@ function start_knative_serving() {
   wait_until_pods_running knative-serving || return 1
 }
 
+# Install Knative Monitoring in the current cluster.
+# Parameters: $1 - Knative Monitoring manifest.
+function start_knative_monitoring() {
+  header "Starting Knative Monitoring"
+  subheader "Installing Knative Monitoring"
+  # namespace istio-system needs to be created first, due to the comment
+  # mentioned in https://github.com/knative/serving/blob/4202efc0dc12052edc0630515b101cbf8068a609/config/monitoring/tracing/zipkin/100-zipkin.yaml#L21
+  kubectl create namespace istio-system
```

Comment:
Please either fail the function or hide the error of the namespace already exists.

Reply:
Done.